docs/src/getting-started/contracts.md

```diff
@@ -8,11 +8,7 @@ Once you've written a contract in Sway and compiled it with `forc build` (read [
 
 Below is how you can deploy your contracts using the SDK. For more details about each component in this process, read [The abigen macro](../abigen/the-abigen-macro.md), [The FuelVM binary file](../contracts/the-fuelvm-binary-file.md), and [The JSON ABI file](../abigen/the-json-abi-file.md).
 
-### The deploy functions
-
-The `Contract::deploy` function is used to deploy a contract binary to the blockhain. To configure the deployment you can use the `DeployConfiguration` struct.
-
-If you are only interested in a single instance of your contract, use the default configuration: `DeployConfiguration::default()`
+First, the `Contract::load_from` function is used to load a contract binary with a `LoadConfiguration`. If you are only interested in a single instance of your contract, use the default configuration: `LoadConfiguration::default()`. After the contract binary is loaded you can use the `deploy()` method to deploy the contract to the blockchain.
 
 ```rust,ignore
 {{#include ../../../examples/contracts/src/lib.rs:deploy_contract}}
```

Comment:
This snippet includes a call to abigen which isn't relevant to what we are describing here. I would move the abigen call into this anchor `use_deployed_contract` since the binding is being used there.
Then just modify the following sentence in docs 

> You can then use the contract methods very simply:

to not say simply and mention abigen.